example/src/main/java/io/netty/example/http/websocketx/benchmarkserver/WebSocketServerHandler.java

```diff
@@ -47,8 +50,13 @@
 
     private static final String WEBSOCKET_PATH = "/websocket";
 
+    private final Set<ChannelHandler> httpHandlers;
     private WebSocketServerHandshaker handshaker;
 
+    public WebSocketServerHandler(Set<ChannelHandler> httpHandlers) {
```

Comment:
And..

- `Iterable` instead of `Set`
- Add another constructor that accepts `ChannelHandler...`?